scripts/consolidate-memory.py

```
@@ -0,0 +1,492 @@
+        description='Consolidate memory logs for pattern detection',
+        epilog='Part of FS1: Agent Memory & Learning System'
+    )
+    parser.add_argument('--days', '-d', type=int, default=7,
+                       help='Number of days to consolidate (default: 7)')
+    parser.add_argument('--dry-run', action='store_true',
+                       help='Show results without writing files')
+    args = parser.parse_args()
+
+    try:
+        memory_dir = get_memory_dir()
+        print(f'\n=== Memory Consolidation ===')
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)